fixups/RegLegacyFixups/InitializeFixup.cpp

```diff
@@ -256,6 +255,63 @@ void InitializeConfiguration()
 
                             specItem.remediationRecords.push_back(recordItem);
                         }
+
+                        // Look for DeletionMarker remediation
+                        else if (type.compare(L"DeletionMarker") == 0)
+                        {
+                            Log("RegLegacyFixups:      is DeletionMarker\n");
```

Comment:
what is with all the spaces?  Are you trying to do a tab?

Reply:
Yes.

Reply:
Please use `\t`

Reply:
Sure, changed.